### ISSUE TITLE
Request Payload's 'Add' Field is sent to server

### ISSUE THREAD

**Issue by nickdaugherty** [ahmadnassri/app-restconsole#61]
When submitting requests, if there is any value inside the 'Request Payload' field for adding new fields (the disabled one), it gets sent to the server. I don't know how a value got in there (I think it's related to saving defaults and then coming back later), but it's in there and I can't remove it. If I inspect the element, the value isn't set, however it appears in the UI and gets sent to server.

At least it should not send that value to the server, but it should also not ever allow a value to get in there.

I can provide a screenshot if needed.

**Comment by ahmadnassri** [comment]
yep, that's no good, I think I came across that and fixed it in the development branch, I'm a few days away from releasing the new version which will include this fix.

cheers.

**Comment by nickdaugherty** [comment]
Great, looking forward to it. I use restconsole on a daily basis, it is outstanding.
